ui/public/index.html

```diff
@@ -3,12 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <meta name="x-public-path-prefix" content="__PUBLIC_PATH_PREFIX__" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/distro-res/favicon.ico" />
```

Comment:
But where is `__DISTRO_STRINGS_RES__`?

Reply:
It is generated when building.

Reply:
I mean seems that there is no source string for replacement?